src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+# Kensetsu Platform
+
+## Theory
+
+### Overall description of the platform
+
+Kensetsu is a lending platform with an overcollateralised stablecoin pegged to the USD. It is a MakerDAO analogue in the SORA ecosystem.
+The native token for the Kensetsu platform is the KEN token, which is used for farming and protocol rewards.
+
+### Key Features - KUSD borrowing (available collateral, parameters). Fee Calculation Process
+
+A Vault is the primary entity which operates on the Kensetsu platform. The Vault has the following parameters;
```

Comment:
One more parameter is `borrow tax` that is 1% now. It is a tax that is applied during borrowing.